example_dags/example_merge.py

```diff
@@ -0,0 +1,92 @@
+import pathlib
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from pandas import DataFrame
+from sqlalchemy import Column, types
+
+from astro import sql as aql
+from astro.files import File
+from astro.sql.table import Metadata, Table
+
+CWD = pathlib.Path(__file__).parent
+DATA_DIR = str(CWD) + "/../tests/data/"
+
+default_args = {
+    "owner": "airflow",
+    "retries": 1,
+    "retry_delay": 0,
+}
+
+dag = DAG(
+    dag_id="example_merge_bigquery",
+    start_date=datetime(2019, 1, 1),
+    max_active_runs=3,
+    schedule_interval=timedelta(minutes=30),
+    default_args=default_args,
+)
+
+
```

Comment:
Don't we need to showcase merge example for all the databases that we support, and usage of Constraints?